File: Java/TextConverter/src/test/java/tddmicroexercises/textconvertor/HtmlTextConverterTest.java
package tddmicroexercises.textconvertor;

import static org.junit.jupiter.api.Assertions.assertEquals;

import org.junit.jupiter.api.Test;

public class HtmlTextConverterTest {

    @Test
    public void foo() {
        HtmlTextConverter converter = new HtmlTextConverter("foo");
        assertEquals("fixme", converter.getFilename());
    }

}

File: kotlin/TextConverter/src/test/kotlin/tddmicroexercises/textconvertor/HtmlTextConverterTest.kt
package tddmicroexercises.textconvertor

import org.junit.jupiter.api.Assertions.assertEquals
import org.junit.jupiter.api.Test

class HtmlTextConverterTest {
    @Test
    fun foo() {
        val converter = HtmlTextConverter("foo")
        assertEquals("fixme", converter.filename)
    }
}
